### PR TITLE
Serial and VJ into Memory Graph - quick fix in Nav Drawer

### LINKED ISSUE #302: Change the VJ and Serial Modes to Memory Graph Mode in the nav drawer
**Describe the bug**
VJ and Serial modes are present in the Navigation Drawer.

**Expected Behavior**
Memory Graph mode should be present instead.

**Would you like to work on the issue?**
Working on it.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #302 
**Changes**: Serial and VJ into Memory Graph

**Screenshot/s for the changes**:

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3293756/app-debug.zip)

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@jddeep I am working on this issue. Doing some other changes as well in the navigation drawer to make it better looking and standard. Kindly close this as it will cause conflicts. I guess it was self-assigned for the same reasons 😕

**Comment by jddeep** [comment]
> @jddeep I am working on this issue. Doing some other changes as well in the navigation drawer to make it better looking. Kindly close this as it will cause conflicts.

@suresh-gandhi As we didn't have a PR for this even after 6 hours. @CloudyPadmal directed to do it asap (on gitter). Anyways, leave improving the UI for navigation drawer for now, keep the changes in the branch of yours, after we finish up with functionalities, you can push that (low priority).

**Comment by jddeep** [comment]
@suresh-gandhi Can you help in testing #296 ? Need it to be tested, to go forward in plotting the data.

**Review by CloudyPadmal — APPROVED** [review]

**Comment by CloudyPadmal** [comment]
@suresh-gandhi but the issue is about the changes made in the PR. What other changes do you suggest?

**Comment by suresh-gandhi** [comment]
@CloudyPadmal I wanted to reciprocate the PSLabs Navigation drawer and was working on the same which I think is more relevant and offers appropriate options. Mario also suggested somewhat along similar lines yesterday. Anyways, it's my mistake I didn't change the issue description accordingly later.

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 1]
```diff
@@ -18,11 +18,8 @@
                     android:layout_height="match_parent"
                     android:title="Relax Mode" />
```
Comment (suresh-gandhi) [inline app/src/main/res/menu/activity_main_drawer.xml]: Remove hardcoding

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 2]
```diff
@@ -18,11 +18,8 @@
...
-                    android:id="@+id/nav_vj"
-                    android:title="VJ Mode" />
-                <item
-                    android:id="@+id/nav_serial"
-                    android:title="Serial Mode" />
+                    android:id="@+id/nav_memory_graph"
+                    android:title="@string/mem_graph" />
                 <item
                     android:id="@+id/nav_connect"
                     android:title="Connect Device" />
```
Comment (suresh-gandhi) [inline app/src/main/res/menu/activity_main_drawer.xml]: here as well and other places in the nav drawer also

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
Removed hardcoding throughout. @CloudyPadmal This can be merged now.

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**